libraries/alicloud_actiontrail_trails.rb

```diff
@@ -16,7 +16,8 @@ class AliCloudActionTrailTrails < AliCloudResourceBase
   def initialize(opts = {})
     # Call the parent class constructor
     super(opts)
-    validate_parameters
+    validate_parameters(required: %i{region})
```

Comment:
please revert this.. as described [here](https://github.com/inspec/inspec-alicloud/pull/53/files#r638990549)

Reply:
Hi @sathish-progress , as above, the region is actually mandatory for some resources (but not all)